src/main/java/edu/cornell/kfs/coa/businessobject/GlobalObjectWithIndirectCostRecoveryAccounts.java

```diff
@@ -0,0 +1,40 @@
+package edu.cornell.kfs.coa.businessobject;
+
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryAccount;
+import org.kuali.rice.krad.bo.GlobalBusinessObjectDetailBase;
+
+public interface GlobalObjectWithIndirectCostRecoveryAccounts {
+
+	public List<IndirectCostRecoveryAccountChange> getIndirectCostRecoveryAccounts();
+
+	public void setIndirectCostRecoveryAccounts(
+			List<IndirectCostRecoveryAccountChange> indirectCostRecoveryAccounts);
+
+	public List<IndirectCostRecoveryAccountChange> getActiveIndirectCostRecoveryAccounts();
+
+	public abstract List<? extends GlobalBusinessObjectDetailBase> getGlobalObjectDetails();
```

Comment:
Not a big deal, but since this is an interface, the public and abstract modifiers are redundant and can be removed.